test/harry/main/org/apache/cassandra/harry/gen/BijectionCache.java

```diff
@@ -124,6 +126,8 @@ public int byteSize()
     @Override
     public int compare(long l, long r)
     {
-        throw new UnsupportedOperationException();
+        T lhs = inflate(l);
```

Comment:
needed for cell reconcile, before we never defined timestamps, but now that we do we need to compare w/e they conflict